activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb

```diff
@@ -632,6 +633,19 @@ def validate_foreign_key(from_table, to_table = nil, **options)
           validate_constraint from_table, fk_name_to_validate
         end
 
+        # Validates the given check constraint.
+        #
+        #   validate_check_constraint :products, name: "price_check"
+        #
+        # The +options+ hash accepts the same keys as SchemaStatements#add_check_constraint.
+        def validate_check_constraint(table_name, **options)
+          return unless supports_validate_constraints?
```

Comment:
I don't think this is necessary, since `supports_validate_constraints?` is always true for the PostgreSQL adapter.

Reply:
@eugeneius I included this to match the implementation of `validate_foreign_key`. Happy to remove it, but it felt better to have those 2 methods match. Let me know what you'd prefer to do!

Reply:
Let's leave it in 👍